che-incubator/che-code
# fix: use ~/.bashrc in sh shell

### What does this PR do?
- takes all the environment variables from the main vscode thread, saves them into `~/.bashrc_variables` file
- before running any task, applies the variables from `~/.bashrc_variables` file

### What issues does this PR fix?

https://github.com/eclipse-che/che/issues/23009

### How to test this PR?
- Create a workspace using repository https://github.com/vitaliy-guliy/vscode-test-extension/tree/test-sh-terminal and the editor image from this PR [quay.io/che-incubator-pull-requests/che-code:pr-383-amd64](https://quay.io/che-incubator-pull-requests/che-code:pr-383-amd64)
- once workspace created, open a terminal and run `export` command
- launch `Run test script` task. Ensure `PATH` variables are equal for both terminals

### Does this PR contain changes that override default upstream Code-OSS behavior?
